ktxspec.adoc

```diff
@@ -1071,6 +1071,21 @@ have shaders that understand this channel layout.
 The bitstream of the UASTC data is described in Chapter 25 _UASTC
 Compressed Texture Image Format_ of <<KDF13>>.
 
+Basis Universal UASTC HDR Format::
+The Universal ASTC HDR image format (UASTC HDR) is indicated by
+`colorModel` `KHR_DF_MODEL_UASTC_HDR` (= 167) together with `vkFormat`
+`VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK` (= 1000066000). The DFD must be
+as described in Section ??? of <<KDF13>>. Images in this format are
+a strict RGB-only ASTC HDR subset that must be transcoded to BC6H
+or decoded to raw 16-bit floating-point values before being uploaded
+to and sampled by a GPU that does not support ASTC HDR. UASTC HDR
+images can be supercompressed with Zstandard (`supercompressionScheme`
+= 2). If supercompression is used, the DFD's `bytesPlane[0-7]`
+must be set to 0 as described in the next subsection.
++
+This color model provides a single channel id `KHR_DF_CHANNEL_UASTC_HDR_RGB`
+(= 0).
```

Comment:
This field is called `channelType`. Also, the `KHR_DF_SAMPLE_DATATYPE_FLOAT` flag must be set.

Reply:
Unfortunately`khr_df.h` calls it CHANNELID

```
KHR_DF_SAMPLEWORD_CHANNELID
```

but I agree we should follow the naming in the Data Format Specification which also uses `channelType` in the non-normative C99 struct for a DFD.

Reply:
I've opened https://github.com/KhronosGroup/dfdutils/issues/14 to rename the `khr_df.h` constants.

Reply:
Resolved in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/0607d1dd466c3836066b1e40768f697791d4b6fd.

Reply:
Given what was pointed out over in https://github.com/KhronosGroup/dfdutils/issues/14 I need to revisit the language changes I made here. Mañana.

Reply:
Looks like KDFS is self-contradictory in that area:

- In Section "5.12. Sample information", the `channelType` has 4 bits and the flags are separate fields.
- In Section "5.15. Sample channelType and qualifiers". the `channelType` is defined as having 8 bits.

Consider reverting the spec updates unrelated to UASTC HDR for now.

Reply:
Reverted in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/65d1cea9646b195061e6b62509acb6190172865c. This also contains improvements to the channel id language.